src/Views/Game.hs

```diff
@@ -120,10 +120,13 @@ addGame api gameID = do
       appendChild gameListDiv gameDiv
 
       clickEventString gameName $ do
-        onServer $ joinGame api <.> gameID
-        players <- onServer $ findPlayersInGame api
-        deleteLobbyDOM
-        createGameDOM api
+        bool <- onServer $ joinGame api <.> gameID
+        case bool of
+          False -> return ()
```

Comment:
Agreed, see #35. Will also create issue for creating a `LobbyMessage` constructor that corresponds to an error.